controllers/api/index.js

```diff
@@ -0,0 +1,6 @@
+const router = require('express').Router();
+
+// /api/twilio/sendsms
```

Comment:
Should this require be set to a variable at the top of the file so that it is obvious what libraries are being used?